### PR TITLE
akon47.ScreenRecorder version 1.1.2

### PR DESCRIPTION
**Pull request by akon47**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/36433)

### CODE REVIEW

**Comment by akon47** [comment]
Why is the "Validation-Executable-Error" label displayed?

File: manifests/a/akon47/ScreenRecorder/1.1.2/akon47.ScreenRecorder.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: akon47.ScreenRecorder
+PackageVersion: 1.1.2
+PackageLocale: en-US
+Publisher: akon47
```
Comment (ItzLevvie) [inline manifests/a/akon47/ScreenRecorder/1.1.2/akon47.ScreenRecorder.locale.en-US.yaml:7]:
```suggestion
Publisher: kimhwan
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/a/akon47/ScreenRecorder/1.1.2/akon47.ScreenRecorder.locale.en-US.yaml (hunk 1 above)
Comment (akon47) [reply manifests/a/akon47/ScreenRecorder/1.1.2/akon47.ScreenRecorder.locale.en-US.yaml]: very very thank you!

**Comment by ItzLevvie** [comment]
> Why is the "Validation-Executable-Error" label displayed?

An engineer will take a look. It might be a false positive.

**Review by ItzLevvie — DISMISSED** [review]

**Comment by ChuckFerring** [comment]
When I attempt to run ScreenRecorder.exe on a clean windows installation nothing launches. Are there any runtime dependencies that may be missing?

**Comment by akon47** [comment]
oh.. I tested it in the sandbox and found the cause.
ScreenRecorder requires "Microsoft.VC++2015-2019Redist-x64".
i added
```
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VC++2015-2019Redist-x64
    MinimumVersion: 14.29.30037.0
```
But that dependency is not installed. Is there any way I can make the dependencies be installed automatically?

**Comment by akon47** [comment]
> When I attempt to run ScreenRecorder.exe on a clean windows installation nothing launches. Are there any runtime dependencies that may be missing?

oh.. I tested it in the sandbox and found the cause.
ScreenRecorder requires "Microsoft.VC++2015-2019Redist-x64".
i added
```
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VC++2015-2019Redist-x64
    MinimumVersion: 14.29.30037.0
```
But that dependency is not installed. Is there any way I can make the dependencies be installed automatically?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> But that dependency is not installed. Is there any way I can make the dependencies be installed automatically?

At the moment, `PackageDependencies` isn't supported yet. It works in the development builds of WinGet where it'll install the VC++ redist, but not in the Azure Pipelines for now.

You will need to include the dependencies in your installer for this PR to pass.

**Pull request closed without merging**